src/riak_pb_ts_codec.erl

```diff
@@ -83,21 +75,21 @@ decode_columns(Columns) ->
 -spec encode_rows(list(list({binary(), ldbvalue()}))) -> [#tsrow{}].
 %% @ignore copied from riakc_ts_put_operator; inverse of make_data
 encode_rows(Measurements) ->
-    rows_for(Measurements, []).
+    lists:map(fun encode_row/1, Measurements).
```

Comment:
OK summary of my conversation with @jonmeredith - "List comprehension is generally preferred but in this case it doesn't really matter"